components/public-api/gitpod/v1/pagination.proto

```diff
@@ -5,17 +5,28 @@ package gitpod.v1;
 option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
 
 message PaginationRequest {
-    // Page size is the maximum number of results to retrieve per page.
-    // Defaults to 25. Maximum 100.
-    int32 page_size = 1;
+  // Page size is the maximum number of results to retrieve per page.
+  // Defaults to 25. Maximum 100.
+  int32 page_size = 1;
```

Comment:
let's deprecate it for now but not remove that you don't need to deal other UIs and services, i.e. add 

```
// use token instead
int32 page_size = 1 [deprecated = true];
```

with the same for total size

Please create and issue in gRPC migration proejct I would take about it. I hope it unblocks you with the configuration work.

Reply:
Will do. In this case I think we wanna keep page_size, but I marked page and total as deprecated.